src/hooks/useWalletConnection.ts

```diff
@@ -112,6 +113,15 @@ export const useWalletConnection = () => {
     key: LocalStorageKey.EvmDerivedAddresses,
     defaultValue: {} as EvmDerivedAddresses,
   });
+  const { ready, authenticated } = usePrivy();
+  const { login } = useLogin({
+    onError: (error) => {
+      if (error !== 'exited_auth_flow') {
```

Comment:
We can add error logging here:
similar to ln242:
`log('useWalletConnection/privy/useLogin', error);`